src/pages/EP_ManageEPTemplate.page

```diff
@@ -89,7 +105,7 @@
                     </div>
                 </apex:outputPanel>
             </form>
-            <p class="slds-text-heading--medium">Manage Engagement Plan Tasks</p>
+            <p class="slds-text-heading--medium slds-p-top--medium">Manage Engagement Plan Tasks</p>
```

Comment:
needs to be a custom label, which reminds me that I gave a bunch of feedback in your original EP pull request that I let go in, since you were gone, but wanted you to address when you returned.  please look at the closed pull request for EP's to see what everything was.  nothing major, or I wouldn't have accepted the merge.  thanks.